pkg/authentication/bootstrapauthenticator/bootstrap.go

```diff
@@ -92,11 +92,11 @@ type BootstrapUserData struct {
 type BootstrapUserDataGetter interface {
 	Get() (data *BootstrapUserData, ok bool, err error)
 	// TODO add a method like:
-	// IsPermanentlyDisabled() bool
 	// and use it to gate the wiring of components related to the bootstrap user.
 	// when the oauth server is running embedded in the kube api server, this method would always
 	// return false because the control plane would not be functional at the time of the check.
 	// when running as an external process, we can assume a functional control plane to perform the check.
```

Comment:
The comment does not make sense withouth just this one line, you'll eventually want to get rid of it all